app/views/home/show.html.erb

```diff
@@ -1,126 +1,144 @@
-<% content_for(:head) do %>
-  <% if homepage_experiment_id %>
-    <!-- Google Analytics Content Experiment code -->
-    <script>function utmx_section(){}function utmx(){}(function(){var
-    k='<%= ENV["HOMEPAGE_EXPERIMENT_ID"] %>',d=document,l=d.location,c=d.cookie;
-    if(l.search.indexOf('utm_expid='+k)>0)return;
-    function f(n){if(c){var i=c.indexOf(n+'=');if(i>-1){var j=c.
-    indexOf(';',i);return escape(c.substring(i+n.length+1,j<0?c.
-    length:j))}}}var x=f('__utmx'),xx=f('__utmxx'),h=l.hash;d.write(
-    '<sc'+'ript src="'+'http'+(l.protocol=='https:'?'s://ssl':
-    '://www')+'.google-analytics.com/ga_exp.js?'+'utmxkey='+k+
-    '&utmx='+(x?x:'')+'&utmxx='+(xx?xx:'')+'&utmxtime='+new Date().
-    valueOf()+(h?'&utmxhash='+escape(h.substr(1)):'')+
-    '" type="text/javascript" charset="utf-8"><\/sc'+'ript>')})();
-    </script><script>utmx('url','A/B');</script>
-    <!-- End of Google Analytics Content Experiment code -->
-  <% end %>
-<% end %>
-
 <% content_for(:page_title, t('service.homepage.title')) %>
 <% content_for(:meta_description, 'A free and impartial government service that helps you understand ' +
                                     'the options for your pension pot.') %>
 
-<p class="home-intro">Phone <b>0800 138 3944</b> to book a free appointment</p>
+<% content_for(:canonical_path, '/') %>
+
+<div class="home-intro l-page-container">
+  <p class="home-intro__text">Phone <b>0800 138 3944</b> to <a href="/appointments" class="home-intro__link">book a free appointment</a></p>
+</div>
+
+<div class="l-grid-row l-home-row l-home-row--accent">
+  <div class="l-page-container">
+    <div class="l-column-two-thirds">
+      <h1 class="l-home-single-column-heading">Welcome to Pension Wise</h1>
+      <p>Free and impartial government guidance to help you understand what you can do with your pension pot.</p>
+    </div>
+    <div class="l-column-third l-home-intro-container">
+      <%= inline_svg "home/folks.svg", class: "l-home-folks" %>
```

Comment:
Some double quoted strings have snuck in here and I feel it's my duty as human linter to inform you of this 😄 🤖